syncano_cli/sync/project.py

```diff
@@ -48,13 +48,34 @@ def update_from_instance(self, instance, all=False, classes=None,
                              scripts=None):
         """Updates project data from instances"""
         LOG.info("Pulling instance data from syncano")
+        classesI = self.classes
         classes = classes or self.classes.keys()
         scripts = scripts or set(s['label'] for s in self.scripts)
         if all:
             classes = None
             scripts = None
         self.classes = pull_classes(instance, classes)
         self.scripts = pull_scripts(instance, scripts)
+        def cmpDicts(d1, d2):
```

Comment:
cmpDicts shouldn't be an inner function.
Also we don't use camelCase in this project.